src/Pane/Pane.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, Ref } from 'react';
+
+import Grid, { GridProps } from '../Grid';
+
+export type Props = GridProps & {
+  innerRef?: Ref<unknown>;
```

Comment:
I left this `Ref<unknown>` because I'm not certain what the type parameter should be and I'm not sure we're even ever using this.  It can be typed properly (or removed) later.